pytorch/inference/docker/1.7.0/py3/cu101/Dockerfile.gpu

```diff
@@ -0,0 +1,137 @@
+FROM nvidia/cuda:10.1-cudnn7-devel-ubuntu16.04
```

Comment:
@ydaiming we should change this: 
`FROM nvidia/cuda:11.0-cudnn8-devel-ubuntu16.04`

Reply:
Changed